src/forwarders.js

```diff
@@ -629,43 +629,48 @@ export default function Forwarders(mpInstance, kitBlocker) {
         }
     };
 
-    // Sideloaded kits are not configured in the UI and do not have kit configurations
-    // They are automatically added to active forwarders.
+    // Unlike UI enabled kits, sideloaded kits are always added to active forwarders.
 
-    // TODO: Sideloading kits currently requires the use of a register method
+    // TODO: Sideloading kits currently require the use of a register method
     // which requires an object on which to be registered.
-    // In the future, when all kits are moved to the config rather than
+    // In the future, when all kits are moved to the mpConfig rather than
     // there being a separate process for MP configured kits and
     // sideloaded kits, this will need to be refactored.
-    this.processSideloadedKits = function(config) {
+    this.processSideloadedKits = function(mpConfig) {
         try {
-            if (Array.isArray(config.sideloadedKits)) {
-                const sideloadedKits = { kits: {} };
-                // First register each kit's constructor onto sideloadedKits,
-                // which is typed { kits: Dictionary<constructor> }.
-                // The constructors are keyed by the name of the kit.
-                config.sideloadedKits.forEach(function(sideloadedKit) {
+            if (Array.isArray(mpConfig.sideloadedKits)) {
+                const registeredSideloadedKits = { kits: {} };
+                const unregisteredSideloadedKits = mpConfig.sideloadedKits;
+                unregisteredSideloadedKits.forEach(function(unregisteredKit) {
                     try {
-                        sideloadedKit.kitInstance.register(sideloadedKits);
+                        // Register each sideloaded kit, which adds a key of the sideloaded kit name
+                        // and a value of the sideloaded kit constructor.
+                        unregisteredKit.kitInstance.register(
+                            registeredSideloadedKits
+                        );
+                        let kitName = unregisteredKit.kitInstance.name;
```

Comment:
I don't see this being reassigned anywhere.

```suggestion
                        const kitName = unregisteredKit.kitInstance.name;
```